src/core/stream.c

```diff
@@ -70,6 +70,8 @@ QuicStreamInitialize(
     Stream->RefCount = 1;
     Stream->SendRequestsTail = &Stream->SendRequests;
     Stream->SendPriority = QUIC_STREAM_PRIORITY_DEFAULT;
+    Stream->ReliableOffsetRecv = 0;
+    Stream->ReliableOffsetSend = 0;
```

Comment:
Unnecessary because of the `CxPlatZeroMemory(Stream, sizeof(QUIC_STREAM));` line above.

Reply:
Notice we only initialize non-zero variables above.